src/server.tsx

```diff
@@ -66,26 +66,37 @@ const buildEpic = async (metadata: EpicMetadata): Promise<ResponseType> => {
     return { html, css };
 };
 
-app.get('/epic', async (req: express.Request, res: express.Response) => {
-    try {
-        const { html, css } = await buildEpic(testData.metadata);
-        const htmlContent = renderHtmlDocument({ html, css });
-        res.send(htmlContent);
-    } catch (error) {
-        console.log('Something went wrong: ', error.message);
-        res.status(500).send({ error: error.message });
-    }
-});
+app.get(
+    '/epic',
+    async (req: express.Request, res: express.Response, next: express.NextFunction) => {
+        try {
+            const { html, css } = await buildEpic(testData.metadata);
+            const htmlContent = renderHtmlDocument({ html, css });
+            res.send(htmlContent);
+        } catch (error) {
+            next(error);
+        }
+    },
+);
+
+app.post(
+    '/epic',
+    async (req: express.Request, res: express.Response, next: express.NextFunction) => {
+        try {
+            const metadata = buildMetadata(req);
+            const { html, css } = await buildEpic(metadata);
+            res.send({ html, css });
+        } catch (error) {
+            next(error);
+        }
+    },
+);
 
-app.post('/epic', async (req: express.Request, res: express.Response) => {
-    try {
-        const metadata = buildMetadata(req);
-        const { html, css } = await buildEpic(metadata);
-        res.send({ html, css });
-    } catch (error) {
-        console.log('Something went wrong: ', error.message);
```

Comment:
I wonder if we care that we lost the `console.log` here? These end up in Kibana via Cloudwatch, which might be helpful for debugging any issues.

Reply:
Yeah I think we lost this unintentionally - I'll bring it back